core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -502,6 +523,14 @@ public AvroCompatibilityLevel compatibilityType() {
     return compatibilityType;
   }
 
+  public boolean useKafkaCoordination() {
+    boolean haveStoreConnectionUrl = !getString(KAFKASTORE_CONNECTION_URL_CONFIG).isEmpty();
+    boolean haveSchemaRegistryConnectionUrl
+        = !getString(SCHEMAREGISTRY_CONNECTION_URL_CONFIG).isEmpty();
```

Comment:
I don't think we need SCHEMAREGISTRY_CONNECTION_URL_CONFIG anymore. If someone doesn't want to use the zookeeper other than that of the Kafka Store, then they should use Kafka Master election algorithm.